docker/generic/start_proxy.py

```diff
@@ -932,6 +932,19 @@ def make_argparser():
         in grpc-json transcoding. This is to support HTML 2.0<https://tools.ietf.org/html/rfc1866#section-8.2.1>.
         Set this flag to true to disable this feature.
         ''')
+
+    parser.add_argument(
+        '--transcoding_match_unregistered_custom_verb', action='store_true',
+        help='''
+        According to the http template[1], the custom verb is **":" LITERAL** at the end of http template.
```

Comment:
nit: can we use backticks \`":" LITERAL \` instead of `**` to signify the code blocks? Using `**` is confusing because `**` is valid syntax in the URI template.

I know it was copied over from flags definition, but this is user-facing so let's make it easier to read

Reply:
Done, this is true for single `*` also, so I changed there too